### ISSUE TITLE
Draw tool recreates drawing on click

### ISSUE THREAD

**Issue by chimosky** [sugarlabs/physics#57]
Steps to reproduce;
* Draw a line after activity opens
* Click anywhere and notice line gets recreated

@quozl was this the intended behaviour?

**Comment by quozl** [comment]
It allows rapid duplication.  The selected tool receives a mouse button up event and constructs the object with whatever was set up during the drag, or if there was no drag then whatever was most recently set up.  For instance, look at the CircleTool class in tools.py.

I don't see any reason to change it; everybody is used to it, and it is easily discoverable.

Note also

> ProTip: If you’ve drawn with a tool before and want multiple copies of the same shape, just do a left-click and the same object with be drawn again. Try it with the Rectangle tool! Draw a square and then left click again, you’ll get another copy of the square.

https://help.sugarlabs.org/en/physics.html

**Comment by chimosky** [comment]
> I don't see any reason to change it; everybody is used to it, and it is easily discoverable.

Agreed, I don't want to change it either, just wanted to know if that was the intended behaviour.
